### DISCUSSION TITLE
Max steps 500

### DISCUSSION THREAD

**Discussion by yjd2007safe** [langgenius/dify#8732] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Is there a limit of Max steps 500 in workflow mode? Is it configurable?

**Comment by kurokobo** [comment]
Refer to: https://github.com/langgenius/dify/issues/8643 https://github.com/langgenius/dify/pull/8644

That limiation will be configurable by environment variables in the next release

**Reply by yjd2007safe** [reply]
thx
